dm/pkg/checker/binlog.go

```diff
@@ -148,8 +148,8 @@ func (pc *MySQLBinlogRowImageChecker) Check(ctx context.Context) *Result {
 		return result
 	}
 
-	// for mysql.version < 5.6.2 || mariadb.version < 10.1.6,  we don't need to check binlog_row_image.
-	if (!IsMariaDB(value) && !version.Ge(mysqlBinlogRowImageRequired)) || (IsMariaDB(value) && !version.Ge(mariadbBinlogRowImageRequired)) {
+	// for mysql.version < 5.6.2,  we don't need to check binlog_row_image.
+	if !version.Ge(mysqlBinlogRowImageRequired) {
```

Comment:
We return a warning in `version check` if it is mariadb. So we needn't check mariadb in `BinlogRowImageChecker`.